reactivesocket-client/src/main/java/io/reactivesocket/client/ClientBuilder.java

```diff
@@ -194,7 +194,7 @@ public void onNext(List<SocketAddress> socketAddresses) {
                         }
                     }
 
-                    List<ReactiveSocketFactory<SocketAddress>> factories =
+                    List<ReactiveSocketFactory<?>> factories =
```

Comment:
The type argument represents the identifier of a server, which is, in our implementation, always `SocketAddress`.
I can refactor the code to remove it (later).